app/views/teachers/schools/_activities_and_actions.html.erb

```diff
@@ -3,10 +3,14 @@
     <%= render 'activity_types/card', activity_type: activity_type, based_on: 'previous activities'%>
   <% end %>
 
```

Comment:
This makes the three activity/programme boxes drop into a single column a bit sooner